### PR TITLE
SA Week 14 Sub1

### PR DESCRIPTION
**Pull request by sabbott7**

### CODE REVIEW

**Comment by halfaro1** [comment]
Closing and reopening the pull request to force the automated check to run...

File: resizeAndAddLogo.py [hunk 1]
```diff
@@ -7,8 +7,39 @@
...
+    if width > height:
+        height = int((SQUARE_FIT_SIZE / width) * height)
+        width = SQUARE_FIT_SIZE
+    else:
+        width = int((SQUARE_FIT_SIZE / height) * width)
+        height = SQUARE_FIT_SIZE
+
+    # Resize the image.
+    print('Resizing %s...' % (filename))
+       im=im.resize((width,height))
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @sabbott7 what's up with this indentation?

Comment (sabbott7) [inline resizeAndAddLogo.py]: Hi Hector,


I think i've sorted the indent problems below.

Now I have a new error on my last commit,... last line 45 (SA Week 14) Sub 16


#TODO: Save changes.

im.save(os.path.join('withLogo', filename))

I get a NameError: name 'im' is not defined...i have "no idea2, why its not picking it up...when im has already been defined earlier ??? Please advise...

Sean
________________________________

File: resizeAndAddLogo.py [hunk 2]
```diff
@@ -7,8 +7,38 @@
 logoIm = Image.open( LOGO_FILENAME )
 logoWidth, logoHeight = logoIm.size
 
+os.makedirs('withLogo', exist_ok=True)
 #TODO: Loop over all files in the working directory
-#TODO: Check if file image needs to be
-#TODO: Calculate the new width and height to resize to.
+for filename in os.listdir('.'):
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @sabbott7 this loop iterates through the files in the current working directory. Remember that the original files live in the `originals` directory, so your program doesn't quite know where to grab the originals from. Thus, the testing script is failing -- it tests for a total of 5 modified files and it currently finds 0. Does this help?

**Comment by halfaro1** [comment]
@sabbott7 
The automated testing for this particular assignment isn't working as expected, so I've graded your project manually. Everything is 👍 on your project, so I am closing this and changing the tag to `graded`. When I transfer these grades to Blackboard, you'll see the tag switch to `grade updated on blackboard`. Great job!

**Pull request closed without merging**